daffodil-test/src/test/resources/org/apache/daffodil/section13/text_number_props/TextNumberProps.tdml

```diff
@@ -4288,4 +4288,26 @@
 
   </tdml:parserTestCase>
 
+  <!--
+     Test Name: textStandardFloatPatternNoSeparators2
+        Schema: textNumberPattern
+          Root: tnp96
+          Purpose: This test shows that when the textNumberPattern contains
+          no grouping/decimal separators, when unparsing, ICU will round the
+          result to match the pattern.
+  -->
+
+  <tdml:unparserTestCase name="textStandardFloatPatternNoSeparators2" root="tnp96" model="textNumberPattern" roundTrip="none">
+
+    <tdml:document>
+      <tdml:documentPart type="text">11</tdml:documentPart>
+    </tdml:document>
+    <tdml:infoset>
+      <tdml:dfdlInfoset>
+        <tnp96>10.9</tnp96>
+      </tdml:dfdlInfoset>
+    </tdml:infoset>
+
+  </tdml:unparserTestCase>
```

Comment:
ICU changed the behavior for how floating point numbers are formatted/unparsed when the textNumberPattern does not contain a decimal separator. The previous behavior was to unparse the number with the separator. New behavior rounds the floating point number. Note that when parsing it will still accept floating point numbers if the pattern does not contain a decimal separator. This means patterns without decimal separators and the data contains a deciaml point may not round trip correct. See textStandardFloatPatternNoSeparators1 above for the parse case.